tracer/build/_build/docker/alpine.dockerfile

```diff
@@ -7,7 +7,15 @@ ENV \
     # Do not generate certificate
     DOTNET_GENERATE_ASPNET_CERTIFICATE=false \
     # Do not show first run text
-    DOTNET_NOLOGO=true \
+    DOTNET_NOLOGO=1 \
+    # Skip first-time experience for quicker startup
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1 \
```

Comment:
I _don't_ think we should set this here. The point of the first-time experience is that dotnet does some things ahead of time, to speed up things for subsequent runs. We pre-build these docker images, so the first-time experience _should_ be run. 

That's why we currently call `dotnet help` after installing .NET, to trigger the first-time experience. If that's not working, we should see about running `dotnet new console` (and removing it again) or something like that

Reply:
Makes sense. Thank you!